### PR TITLE
Spring cleaning

### PR DESCRIPTION
**Pull request by gdpelican**
Some quick refactoring I did on the plane wee! (Don't merge me yet)

### CODE REVIEW

File: app/controllers/users/sessions_controller.rb [hunk 1]
```diff
@@ -15,23 +15,31 @@ class Users::SessionsController < Devise::SessionsController
 
   # and this
   def create_parse_user_if_needed
-    if ENV['PARSE_ID'] and request.subdomain == ENV['PARSE_SUBDOMAIN']
```
Comment (robguthrie) [inline app/controllers/users/sessions_controller.rb]: This is one thing I don't think you should ever touch. We'll probably remove it soon.. and it was only added for a single group.. and there are no tests.

Comment (robguthrie) [inline app/controllers/users/sessions_controller.rb]: that aside.. looks like a nice refactor

**Comment by gdpelican** [comment]
I want to hold off on this for now until I get a better chance to test out the discussion cache refactor

**Comment by gdpelican** [comment]
Hey @robguthrie how do you feel about merging this after the tests go green?

**Comment by robguthrie** [comment]
good!

**Pull request merged**